src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -162,11 +162,11 @@ public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList
             out bool hasAccessorList,
             out bool accessorsHaveImplementation,
             out bool isInitOnly,
-            out CSharpSyntaxNode? getSyntax,
-            out CSharpSyntaxNode? setSyntax)
+            out AccessorDeclarationSyntax? getSyntax,
+            out AccessorDeclarationSyntax? setSyntax)
         {
             var syntax = (BasePropertyDeclarationSyntax)syntaxNode;
-            isAutoProperty = true;
+            isAutoProperty = false;
```

Comment:
>isAutoProperty [](http://example.com/codeflow?start=12&length=14)

Can we revert changes around ```isAutoProperty``` local for the purpose of the scoped down PR?

Reply:
Done